src/path_specific.jl

```diff
@@ -1,32 +1,52 @@
 using Omega, Random
+using LightGraphs:SimpleEdge
+
 import Omega:intervene, Interventions
 
 export apply_ps_intervention, BlockedEdges
 
-function apply_ps_intervention(model::CausalModel, x₁::Intervention, blocked_edges::AbstractVector{Bool}, x₂::Intervention, ω::AbstractΩ)
+struct BlockedEdges
+    edges::Vector{Edge}
```

Comment:
`edges` should also be parametrised by a concrete type. Ref: https://docs.julialang.org/en/v1/manual/performance-tips/#man-performance-abstract-container